test/CreatorToken.t.sol

```diff
@@ -49,6 +49,44 @@ contract CreatorTokenTest is Test {
     (uint256 _creatorFee, uint256 _adminFee) = creatorToken.calculateFees(_price);
     return _price + _creatorFee + _adminFee;
   }
+
+  function buyAToken(address _buyer) public {
+    (uint256 _tokenPrice, uint256 _creatorFee, uint256 _adminFee) = creatorToken.nextBuyPrice();
+    uint256 _totalPrice = _tokenPrice + _creatorFee + _adminFee;
+    uint256 originalCreatorTokenBalance = payToken.balanceOf(address(creatorToken));
+    uint256 originalBuyerBalance = creatorToken.balanceOf(_buyer);
+    uint256 originalTotalSupply = creatorToken.totalSupply();
```

Comment:
Similarly, consider: `_originalCreatorTokenSupply`

Reply:
Updated in https://github.com/showtime-xyz/creator-tokens/pull/14/commits/89f1666530442849e5fcc6a35261447ee2cf1cbf